hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/tasks/TestOmTableInsightTask.java

```diff
@@ -111,11 +116,38 @@ public void testReprocess() throws Exception {
     assertEquals(5L, getCountForTable(DELETED_TABLE));
   }
 
+
   @Test
-  public void testProcess() {
+  public void testReprocessForSize() throws Exception {
+    // Populate the OpenKeys table in OM DB
+    writeOpenKeyToOm(reconOMMetadataManager,
+        "key1", "Bucket1", "Volume1", null, 1L);
+    writeOpenKeyToOm(reconOMMetadataManager,
+        "key1", "Bucket2", "Volume2", null, 2L);
+    writeOpenKeyToOm(reconOMMetadataManager,
+        "key1", "Bucket3", "Volume3", null, 3L);
+
+    // Populate the OpenFile table in OM DB
+    writeOpenFileToOm(reconOMMetadataManager,
+        "file1", "Bucket1", "Volume1", "file1", 1, 0, 1, 1, null, 1L);
+    writeOpenFileToOm(reconOMMetadataManager,
+        "file2", "Bucket2", "Volume2", "file2", 2, 0, 2, 2, null, 2L);
+    writeOpenFileToOm(reconOMMetadataManager,
+        "file3", "Bucket3", "Volume3", "file3", 3, 0, 3, 3, null, 3L);
+
+    Pair<String, Boolean> result =
+        omTableInsightTask.reprocess(reconOMMetadataManager);
+    assertTrue(result.getRight());
+    assertEquals(6L, getSizeForTable(OPEN_KEY_TABLE));
+    assertEquals(6L, getSizeForTable(OPEN_FILE_TABLE));
```

Comment:
can we do some assertions for counts also for both openKey and openFile tables ?

Reply:
I have done the counts for both in the method! could you please provide more details on the specific types of assertions we need to include in the test?